doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,72 @@
+Traffic Shaping
+===============
+
+Linux offers a very rich set of tools for managing and manipulating the transmission of packets. And
+since the host system is Linux based, those tools can be used to manage and shape traffics on Software
+Containers. Linux community is very familiar with the tools available under Linux for packet mangling and
+firewalling (netfilter, and before that, ipchains) as well as hundreds of network services which can
+run on the operating system.
+
+Linux community are aware of the tremendous power of the traffic control subsystem which has grown
+and matured. To glace those variaty of solutions the linux documentation project would be very good
+resource to start. Please check `TLDP <http://www.tldp.org/HOWTO/html_single/Traffic-Control-HOWTO/#intro>`_
+for having understanding of the concept. `Linux Advanced Routing & Traffic Control <http://lartc.org/howto/>`_
+is another good source that presents deep information about the topic.
+
+However there are many ways of shaping the traffic, the purpose of this document is to give brief information
```

Comment:
Can we phrase this so that we don't talk about "purpose of this document", but rather "what we describe here is..."